### ISSUE TITLE
cwSet problem

### ISSUE THREAD

**Issue by ymintx** [fiote/ToS-Addons#9]
After several test attempt.

It seem if your Body and Pant part leave empty and create a set (lets name it SetEmpty), whenever u try to run the SetEmpty with Body and Pant parts equipped beforehand, it will remove Haircostume-1.

**Comment by Crevox** [comment]
Yeah, this happens to me too. It removes my hair costume 1 and/or reequips it often, and not even sure if it's because of these conditions.

I have to run a set load twice to ensure it equips it.

EDIT: If I load another set and that set has similar pieces, then it unequips my hair costume 1.
